operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md

```diff
@@ -17,15 +17,14 @@ Please provide the following for Mattermost by email:
 
 If you provide screenshots, please help ensure they meet our [screenshot development guidelines](https://handbook.mattermost.com/operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs).
 
-A link to the final draft of the blog post will be sent to you by email, so that you can offer feedback and recommendations on it.
+A link to the final draft of the blog post will be sent to you by email prior to publication, so that you can offer feedback and recommendations on it.
```

Comment:
@aedott, please confirm that this is done prior to publication. Thanks!

Reply:
Yes, all contributors have a chance to review the final draft before it goes live!